src/lib.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Please add `#[deny(missing-docs)]` and comment the public interface.

Reply:
Fixed as well.